app/src/components/home_button.js

```diff
@@ -0,0 +1,7 @@
+export default function Button({text}) {
```

Comment:
This function should be named HomeButton to match the file name

Reply:
Sounds good, on it.